python/ray/tune/tests/test_logger.py

```diff
@@ -169,8 +169,11 @@ def testTBX(self):
                     "D": 123
                 }
             },
-            "d": np.int64(1),
-            "e": np.bool8(True)
+            "int32": np.int32(1),
+            "int64": np.int64(1),
+            "bool8": np.bool8(True),
+            "float32": np.float32(1),
+            "float64": np.float64(1),
```

Comment:
Do we test/check the logged output somewhere?

Reply:
I just added a test, take a look?